### ISSUE TITLE
[Feature Request] Add share to Kuroba

### ISSUE THREAD

**Issue by Astridchan** [Adamantcheese/Kuroba#285]
From an anon who is too lazy to make a GitHub account:
>PLS ADD SHARE TO KUROBA in the same vein as Dashchan
>how does it work? you can share to from any app in the phone and chose Dashchan
>depending of what you share, dashchan adds it to your reply draft, either a picture, a webm, text...
>here is the documentation I have found to do it: 
>https://developer.android.com/training/sharing/receive
>there is even code for each type on file you share
>It is a very useful functionality instead of copying and pasting text, or having to find a picture on the shit file selector

I will add that the way it works in Dashchan is that if you share to the app with a thread open, whatever you're sharing is added to the reply draft, and if you haven't got a thread open it gets added to the new thread draft.

**Comment by Adamantcheese** [comment]
I mean, you can already copy/paste text into a draft and use an image URL to add an image, but alright.

**Comment by Astridchan** [comment]
In the case of text, true, not much difference from just copy pasting, but I can see it being faster/more convenient for sharing locally saved media straight from a file manager / gallery app rather than going through Kuroba's file picker.

**Comment by Pentaphon** [comment]
@Astridchan I agree. Kuroba/Clover has needed this for quite some time to make it feel like every other Android app that appears on the share menu.

**Comment by ghost** [comment]
+1 on this, plus accepting images from the clipboard. Image URL isn't always convenient. Requires overwriting your current clipboard, and isn't always accessible. booru.soy for example automatically downloads images when you open in a new tab to get the URL, and my phone or browser does not support copying the image URL any other way. Also more convenient for sharing images directly from non-browser apps.

**Comment by Adamantcheese** [comment]
Chrome: Hold down, preview image, hit the thing to pop it into a new tab, copy the address.
Firefox: Hold down, copy address.
Anything else; Probably similar to the two above.
I understand that it's more convenient to share to the application, but there's your workaround for now. As always, PRs are welcome.
